ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -0,0 +1,148 @@
+''' EmailAgent
+
+  This agent sends an email every 30 minutes with aggregated actions taken for each site
+
+'''
+
+
+import os
+import json
+from datetime import datetime
+from DIRAC                                                       import S_OK, S_ERROR
+from DIRAC.Core.Base.AgentModule                                 import AgentModule
+from DIRAC.ResourceStatusSystem.Utilities                        import RssConfiguration
+from DIRAC.Interfaces.API.DiracAdmin                             import DiracAdmin
+
+__RCSID__ = '$Id: $'
+
+AGENT_NAME = 'ResourceStatus/EmailAgent'
+
+class EmailAgent( AgentModule ):
+
+  def __init__( self, *args, **kwargs ):
+
+    AgentModule.__init__( self, *args, **kwargs )
+    self.diracAdmin = DiracAdmin()
+
+    self.default_value = None
+    self.dirac_path = None
+    self.dirac_path = None
+    self.cacheFile = None
+
+  def initialize( self ):
+    ''' EmailAgent initialization
+    '''
+
+    self.default_value = '/opt/dirac/pro/work/ResourceStatus/'
```

Comment:
I'm using an environmental variable in the next line, this is just a `default_value` as I have written that is going to be used in case the environment variable `DIRAC` does not exists.